Add/Readd regtest artworks and adapt it in all files

cevap opened the pull request:
ref #782 

I splitted this pull from #782 , in pull #782 snap icon for the shortcut to launch regtest is added, in this pull QT should 

If PR #782 is not accepted, I guess this PR should be rejected too.

Regtest icon/png (as you like pink, I colorized current testnet icon with pink to create regtest icon):
![bitcoin_regtest](https://user-images.githubusercontent.com/24996551/52409239-352c3100-2ad5-11e9-9dad-9b4eef8b6bbf.png)

Regtest splashscreen (I simply inverted current testnet image as you probably will have own design):
![splash_regtest](https://user-images.githubusercontent.com/24996551/52409293-6573cf80-2ad5-11e9-9ed4-8e35dbac7921.png)

Testing of this PR:
![grafik](https://user-images.githubusercontent.com/24996551/52415506-6b72ac00-2ae7-11e9-9071-27456a6e9a9e.png)

![grafik](https://user-images.githubusercontent.com/24996551/52415530-7d544f00-2ae7-11e9-9f71-0270089049d7.png)

[debug.log](https://github.com/PIVX-Project/PIVX/files/2841074/debug.log)

Fuzzbawls commented:
Appreciate this. We do have a complete  regtest update private branch that includes icon/image updates which will probably supersede this, but good looking out

nashanas commented on share/qt/make_windows_icon.sh:
```diff
@@ -15,3 +15,11 @@ convert ${ICON_SRC} -resize 32x32 bitcoin-32.png
 convert bitcoin-16.png bitcoin-32.png bitcoin-48.png ${ICON_DST}
 rm bitcoin-16.png bitcoin-32.png bitcoin-48.png
+#regtest
+ICON_SRC=../../src/qt/res/icons/bitcoin_regtest.png
```
is not better to change name to pivx_regtest.png?

cevap replied:
sure it can be, just holded on the naming convention already used, using pivx_regtest would break it if bitcoin and bitcoin_testnet are used. I would either change them all or keep them as they are. Intention of making this PR was providing launcher for regtest with snap, I was not aware that pivx will/wants to keep regtest just wanted to have icon for regtest installed which differs from current testnet/main net icons/splashscreens.

nashanas commented on src/qt/pivx.qrc:
```diff
@@ -59,6 +59,7 @@
         <file alias="explorer">res/icons/explorer.png</file>
         <file alias="automint_inactive">res/icons/automint_inactive.png</file>
         <file alias="onion">res/icons/onion.png</file>
+        <file alias="bitcoin_regtest">res/icons/bitcoin_regtest.png</file>
```
change bitcoin_regtest to pivx_regtest

cevap replied:
I guess here the same answer applies: https://github.com/PIVX-Project/PIVX/pull/809#discussion_r254857699

cevap commented:
> Appreciate this. We do have a complete regtest update private branch that includes icon/image updates which will probably supersede this, but good looking out

I was not aware if pivx keeps regtest at all, like in first comment stated, as I added icon for starting on snapcraft, then if it would be merged, it would be probably good idea to have current PR too. @nashanas suggested to change naming of files which can be done here, this PR is open for authoring.

As I did add new icon, I did run optimization script for pngs and saw that one of pngs was not optimized, its not bad having this script and running on other sources like wiki etc, thats why I made #808 .

When I tested snap, it failed to build for s390x and ppc64el, so I added additional patches here #810 

But all in all, it is actually about snapcraft and intention of currrent PR is/was not naming convention, I guess if decided to change, this can be done during the merge process.

Fuzzbawls commented:
This is covered in #812

blondfrogs commented:
Can be closed: Functionality is added in #812
